app/src/main/java/org/fossasia/openevent/app/contract/view/LoginView.java

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.app.contract.view;
+
+public interface LoginView {
+
+    void showProgressBar(boolean show);
```

Comment:
It should be implemented by View obviously as it will be called by presenter when need of doing an asynchronous task